frontend/src/containers/DocumentList.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
?

Reply:
Pfff. This must have been added somehow automatically by the editor. Thanks. Removed